src/FlyCube/CommandList/VKCommandList.cpp

```diff
@@ -766,6 +766,22 @@ void VKCommandList::ResolveQueryData(const std::shared_ptr<QueryHeap>& query_hea
                                          vk::QueryResultFlagBits::eWait);
 }
 
+void VKCommandList::SetGraphicsConstant(uint32_t root_parameter_index, uint32_t value, uint32_t byte_offset)
+{
+    decltype(auto) pipeline_layout = m_state->GetPipelineLayout();
+    m_command_list->pushConstants(pipeline_layout,
```

Comment:
You also need to specify `pipeline_layout_info.pPushConstantRanges` in VKBindingSetLayout. But it probably won't work without information about push constant size.